frontend/src/common/queries/wheresMyGene.ts

```diff
@@ -672,13 +910,6 @@ interface Dataset extends RawDataset {
   label: string;
 }
 
-export const EMPTY_FILTERS: State["selectedFilters"] = {
```

Comment:
moved this to `reducer.ts`, since it caused circular dependency and broke the FE build